<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>marketDataSourceCfg</class>
 <widget class="QWidget" name="marketDataSourceCfg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>marketDataSourceCfg</string>
  </property>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>监听IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip0">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>监听端口</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>源名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>绑定网卡</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_nic">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>lineEdit_ip0</tabstop>
  <tabstop>lineEdit_ip1</tabstop>
  <tabstop>lineEdit_ip2</tabstop>
  <tabstop>lineEdit_ip3</tabstop>
  <tabstop>lineEdit_port</tabstop>
  <tabstop>comboBox_type</tabstop>
  <tabstop>comboBox_nic</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
